runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -914,6 +918,16 @@ fn ecrecover_base(ctx: &mut EstimatorContext) -> GasCost {
     fn_cost(ctx, "ecrecover_10k", ExtCosts::ecrecover_base, 10_000)
 }
 
+#[cfg(feature = "protocol_feature_ed25519_verify")]
+fn ed25519_verify_base(ctx: &mut EstimatorContext) -> GasCost {
+    fn_cost(ctx, "ed25519_verify_10k", ExtCosts::ed25519_verify_base, 10_000)
+}
+
+#[cfg(feature = "protocol_feature_ed25519_verify")]
+fn ed25519_verify_byte(ctx: &mut EstimatorContext) -> GasCost {
+    fn_cost(ctx, "ed25519_verify_10k", ExtCosts::ed25519_verify_byte, 10_000)
+}
+
```

Comment:
It's a bit odd that both parameters use the exact same estimation. ;)

But as stated already, we can take care of the estimation separately. For now, please add a comment that these are only placeholders.